### ISSUE TITLE
Configure Alarms and the time from the presentation 

### ISSUE THREAD

**Issue by edvm** [edvm/pysenteishon#12]
Pysenteishon should have a time clock that let the user configure how much will be the presentation duration. Ideally will be a countdown clock. Ideally we should let the user configure 'alarms', for example raise an event like make the phone vibrate when only 15 minutes are left to finish the presentation.

**Comment by humitos** [comment]
This is done in some way. At least, right now there is a count down that I can set by myself.

Also, the browser ask me for permissions to notifications. But, I think they are not raised at all...

**Comment by edvm** [comment]
Yes @humitos , @pardo is working on this (to make notifications be visible) :)
